content/algorithms/gossip_sub.md

```diff
@@ -5,7 +5,7 @@ dashboardWeight: 2.0
 dashboardState: stable
 dashboardTests: 0
 dashboardAudit: done
-dashboardAuditURL: /#section-appendix.audit_reports.2020-06-03-gossipsub-design-and-implementation
+dashboardAuditURL: /#section-appendix.audit_reports.gossipsub
 dashboardAuditDate: '2020-06-03'
```

Comment:
the problem is that it will be out of date with the audit section quickly, we either use the audit section content as the source of truth or the frontmatter properties.